docs/application-development-guide.md

```diff
@@ -2904,6 +3294,30 @@ Content is not available
 
 Content is not available
 
+</TabItem>
+<TabItem value="python">
+
+To define a Signal, set the Signal decorator `@workflow.signal` on the Signal function inside your Workflow.
+
+```python
+@workflow.signal
```

Comment:
Decorators should be the same indention as the functions they're decorating. You may also want to make it clear in the snippet this is a method and not a top-level function as it may appear here.